docs/cloud-messaging/first-message.md

```diff
@@ -73,6 +73,77 @@ which records the number of messages sent and opened on Apple and Android
 devices, along with data for "impressions" (notifications seen by users) for
 Android apps.
 
+## Handling Interaction
+
+Since notifications are a visible cue, it is common for users to interact with it (by pressing them).
```

Comment:
I *think* that some special-accessibility interfaces allow interaction with non-visible (specifically audio-rendered) notification widgets, so this entire sentence may be best to omit.

Suggest starting with just "Default notification behavior on both Android and . . . "